code/modules/mob/living/carbon/human/species/shadow.dm

```diff
@@ -0,0 +1,25 @@
+/datum/species/shadow
+	name = "Shadow"
+
+	icobase = 'icons/mob/human_races/r_shadow.dmi'
+	deform = 'icons/mob/human_races/r_shadow.dmi'
+
+	default_language = "Galactic Common"
+	unarmed_type = /datum/unarmed_attack/claws
```

Comment:
Is there a particular reason they have claws? I don't think that was a thing before.